dirjump

```diff
@@ -111,12 +125,17 @@ dirjump()
 	else
 		echo "dirjump: the requested dir path number is out of range"
 	fi
+	if [[ -f "$last_del" ]]
+	then
+        last_del
+        fi
```

Comment:
These 4 lines should be placed below `print_directory_history` (line 108) to make it works.

And a little indentation fix:
```suggestion
      fi
```